azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/models/CallCompositeLocalOptions.java

```diff
@@ -69,24 +64,23 @@ public CallCompositeParticipantViewData getParticipantViewData() {
      * Set a {@link CallCompositeParticipantViewData} to be used
      * @param participantViewData
      */
-    public void setParticipantViewData(@Nullable final CallCompositeParticipantViewData participantViewData) {
+    public void setParticipantViewData(final CallCompositeParticipantViewData participantViewData) {
```

Comment:
Let's return from setter CallCompositeLocalOptions instead of void

Reply:
Fixed in [a2f7d8a](https://github.com/Azure/communication-ui-library-android/pull/395/commits/a2f7d8ad9b5474e99f9f567e800ca736a339d646)